chrome-launcher.ts

```diff
@@ -12,16 +12,16 @@ import * as rimraf from 'rimraf';
 import * as chromeFinder from './chrome-finder';
 import {getRandomPort} from './random-port';
 import {DEFAULT_FLAGS} from './flags';
-import {makeTmpDir, defaults, delay} from './utils';
+import {makeTmpDir, defaults, delay, getPlatform, toWinDirFormatIfNeeded} from './utils';
 const log = require('lighthouse-logger');
 const spawn = childProcess.spawn;
 const execSync = childProcess.execSync;
 const isWindows = process.platform === 'win32';
```

Comment:
use getPlatform() instead. also on L134 and L150

Reply:
Done